apps/fz_http/priv/repo/migrations/20220530073152_create_mfa_methods.exs

```diff
@@ -0,0 +1,25 @@
+defmodule FzHttp.Repo.Migrations.CreateMfaMethods do
+  use Ecto.Migration
+
+  @create_query "CREATE TYPE mfa_type_enum AS ENUM ('totp', 'native', 'portable')"
+  @drop_query "DROP TYPE mfa_type_enum"
+
+  def change do
+    execute(@create_query, @drop_query)
+
+    create table("mfa_methods", primary_key: false) do
+      add :id, :binary_id, primary_key: true
```

Comment:
Is this for UUID? Yeah was thinking we should probably use UUIDs going forward for all primary keys -- we may need to identify user/devices/rules in different DBs soon.

Reply:
Yes it's for using uuid as pk. (And skipped serial on the above line)